subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/CrossVersionPerformanceTestRunner.groovy

```diff
@@ -277,6 +277,7 @@ class CrossVersionPerformanceTestRunner extends PerformanceTestSpec {
             .warmUpCount(warmUpRuns)
             .invocationCount(runs)
             .buildMutators(buildMutators)
+            .multiVersion(true)
```

Comment:
What does `multiVersion` mean here? Does it mean `crossVersion`? If so why do we change it to another terminology?

Reply:
Yeah, you are right. I should rename the method to `crossVersion`.